### ISSUE TITLE
Regression in verifyOrder in version 1.10.2

### ISSUE THREAD

**Issue by jakob-grabner** [mockk/mockk#507]
I noticed a test failed after I updated to 1.10.2 in a android project

I use a verify order clause as follows

```
 verifyOrder {
                tracker.track(any(), "play", "param", any())
                tracker.track(any(), "pause", "param", any())
                tracker.track(any(), "play", "param", any())
                tracker.track(any(), "pause", "param", any())
                tracker.track(any(), "play", "param", any())
 }
```

### Expected Behavior

Test still works after update to 1.10.2

### Current Behavior
Test fails with exception:

```
java.lang.AssertionError: Verification failed: calls are not in verification order

Matchers: 
+Tracker(#123).track(any(), eq(play), eq(param), any()))
Tracker(#123).track(any(), eq(pause), eq(param), eq({})))
Tracker(#123).track(any(), eq(play), eq(param), eq({})))
Tracker(#123).track(any(), eq(pause), eq(param), eq({})))
Tracker(#123).track(any(), eq(play), eq(param), eq({})))

Calls:
1) Tracker(#123).report(Context(#131), start)
4) +Tracker(#123).track(Application(#144), play, param, {some_key=some_value})
5) Tracker(#123).track(Application(#144), pause, param, {some_key=some_value})
6) +Tracker(#123).track(Application(#144), play, param, {some_key=some_value})
7) Tracker(#123).track(Application(#144), pause, param, {some_key=some_value})
8) +Tracker(#123).track(Application(#144), play, param, {some_key=some_value})
```

### Failure Information (for bugs)

Looks like `eq` is used instead of `any` for all but the first verify call

**Comment by Endhuine** [comment]
This regression seems to be solved from 1.10.3.

**Comment by Raibaz** [comment]
Nice, thanks for checking.
